README.md

```diff
@@ -357,6 +357,28 @@ relationships: {
 }
 ```
 
+## Serialize mixed data (dynamic type)
+
+If your data contains one or multiple object of varying type, It's possible to define a configuration object instead of the type-string as the first argument of ```serialize``` and ```serializeAsync``` function with these options :
```

Comment:
Typos:
- one or multiple object**s** of varying type**s**
- **i**t's possible (*it* lowercased)
- maybe remove the `function` in the end
- options: (no space before `:`)